src/main/java/htsjdk/samtools/SAMRecord.java

```diff
@@ -1116,6 +1116,10 @@ public void setValidationStringency(final ValidationStringency validationStringe
         this.mValidationStringency = validationStringency;
     }
 
+    public boolean hasAttribute(final String tag) {
```

Comment:
needs javadoc

Reply:
done. Thought the name is very obvious...